source/configuring-ember/specifying-url-type.md

```diff
@@ -1,4 +1,12 @@
-By default the Router uses the browser's hash to load the starting state of your
+The Ember router has four options to manage your application's URL: `auto`
+(default), `hash`, `history`, and `none`. By default, Ember CLI configures the
```

Comment:
What about something like the following?

> `auto` which gracefully degrades, and `none`.

I'm otherwise :+1: with this idea.